### ISSUE TITLE
Warning printed after migrating to (joinfaces 5/springboot 3)

### ISSUE THREAD

**Issue by mtbadi39** [joinfaces/joinfaces#1619]
After migrating a project to Spring Boot v3.0 / Joinfaces v5.0.0, the following warnings are printed out :

```
WARN 15516 --- [  restartedMain] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.joinfaces.autoconfigure.scopemapping.CdiScopeAnnotationsAutoConfiguration
WARN 15516 --- [  restartedMain] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.joinfaces.autoconfigure.scopemapping.FacesScopeAnnotationsAutoConfiguration
WARN 15516 --- [  restartedMain] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.joinfaces.autoconfigure.scopemapping.Jsf3ScopeAnnotationsAutoConfiguration
WARN 15516 --- [  restartedMain] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.joinfaces.autoconfigure.faces.JakartaFaces3AutoConfiguration
WARN 15516 --- [  restartedMain] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.joinfaces.autoconfigure.javaxfaces.JsfBeansAutoConfiguration$Jsf2_3AutoConfiguration
WARN 15516 --- [  restartedMain] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.joinfaces.autoconfigure.faces.FacesServletAutoConfiguration
WARN 15516 --- [  restartedMain] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.joinfaces.autoconfigure.primefaces.PrimefacesFileUploadServletContextAutoConfiguration
WARN 15516 --- [  restartedMain] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.joinfaces.autoconfigure.servlet.initparams.ServletContextInitParameterPropertiesAutoConfiguration
```

from [Spring-Framework-6 Migration Guide](https://github.com/spring-projects/spring-framework/wiki/Upgrading-to-Spring-Framework-6.x) :

> **LocalVariableTableParameterNameDiscoverer** is deprecated now and logs a warning for each successful resolution attempt (it only kicks in when **StandardReflectionParameterNameDiscoverer** has not found names). Compile your Java sources with the common Java 8+ **-parameters** flag for parameter name retention (instead of relying on the **-debug** compiler flag) in order to avoid that warning, or report it to the maintainers of the affected code. With the Kotlin compiler, we recommend the **-java-parameters** flag for completeness.

I have added **-parameters** flag to **maven-compiler-plugin** but no luck !
```
<compilerArgs>
    <arg>-parameters</arg>
</compilerArgs>
```

**Comment by larsgrefer** [comment]
This should be fixed in 10368f44a040be5fae394cead726366771c2de00

**Comment by mtbadi39** [comment]
fixed in [5.0.1](https://github.com/joinfaces/joinfaces/releases/tag/5.0.1)

thanks a lot @larsgrefer
